src/models/installation.ts

```diff
@@ -94,7 +95,7 @@ export default class Installation extends Sequelize.Model {
 		await this.destroy();
 	}
 
-	async subscriptions(): Promise<Subscription[]> {
+	async subscriptions(): Promise<SubscriptionClass[]> {
```

Comment:
I wonder what's the difference between Subscription and SubscriptionClass? And how does this type change affect things?

Reply:
yeah, we need to fix that at some point.  It's because we're using an export default class called `Subscription` at `src/models/subscription` but we also have a export const value called `Subscription` which is an instance of the class that's being built by Sequelize and provided by `src/models/index`.  Values and Class types are different and you cannot place a value as a type of a function.  If you look at the imports at the top of the file, I had to import both the class type and the value as you need to refer to the value if you want to do operations on the DB with sequelize or else it'll just error at runtime.